pymc3/stats.py

```diff
@@ -482,7 +482,7 @@ def bpic(trace, model=None):
     return 3 * mean_deviance - 2 * deviance_at_mean
 
 
-def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
+def compare(model_dict, ic='WAIC', method='stacking', b_samples=1000,
```

Comment:
maybe you can change the name to `models_dict`...

Reply:
`model_dict` is short for "model dictionary"; adjectives don't have plural forms in English.

Reply:
OK, I was thinking of "dictionary of models"